inc/dropdown.class.php

```diff
@@ -1839,7 +1839,7 @@ static function showFromArray($name, array $elements, $options = []) {
 
       $param['option_tooltips'] = Html::entities_deep($param['option_tooltips']);
 
-      if ($param["display_emptychoice"]) {
+      if ($param["display_emptychoice"] && ($param['noselect2'] || !$param["multiple"])) {
```

Comment:
ok, last comment for me, we should indicate in dockblock that using `multiple` parameter is incompatible with `display_emptychoice`

Apart this last, PR ok for me

Reply:
Done.